packages/core/src/external/carequality/dq/create-metadata-xml.ts

```diff
@@ -54,7 +54,8 @@ export function createExtrinsicObjectXml({
     healthcareFacilityTypeCode?.coding?.[0]?.code || DEFAULT_HEALTHCARE_FACILITY_TYPE_CODE_NODE;
 
   const organizationName = organization?.name || ORGANIZATION_NAME_DEFAULT;
-  const organizationId = organization?.id || METRIPORT_HOME_COMMUNITY_ID_NO_PREFIX;
+  const organizationId =
+    organization?.identifier?.[0]?.value || METRIPORT_HOME_COMMUNITY_ID_NO_PREFIX;
```

Comment:
is there a way we can explicitly check for the OID here? array indexing makes things brittle in case we ever add something to  `organization?.identifier?` in the future

Reply:
Updated it to find an identifier which starts with our OID